src/app/pages/products-list/products-list.component.html

```diff
@@ -24,37 +24,64 @@
     <app-card class="card" [product]="productsFromNgIf[4]" (buy)="onProductBuy($event)"></app-card> -->
 
     <!-- Раскомментрировать для ДЗ -->
-    <!-- <ng-container
+    <ng-container
         *appPagination="
             let productsGroup of products;
+            let appPaginationOf = appPaginationOf;
             chankSize: 4;
-            let pageIndexes = pageIndexes;
+            let pageIndex = pageIndex;
+            let productsFromNgIf = productsFromNgIf;
             let activeIndex = index;
             let next = next;
             let back = back;
+            let selectCard = selectCard;
             let selectIndex = selectIndex
         "
-    > -->
-    <div class="cards">
-        <app-card
-            *ngFor="
-                let product of productsFromNgIf;
-                trackBy: trackById;
-                let index = index;
-                let first = first;
-                let last = last;
-                let even = even;
-                let odd = odd
-            "
-            class="card"
-            [product]="product"
-            (buy)="onProductBuy($event)"
-        ></app-card>
-    </div>
-    <div class="pagination">
-        <!-- Add pagination control -->
-    </div>
-    <!-- </ng-container> -->
+    >
+        <!-- <ng-container #cards></ng-container> -->
+        <div class="pagination__row">
+            <button
+                mat-icon-button
+                color="primary"
+                type="button"
+                class="pagination__button-prev"
+                (click)="back(); $event.stopPropagation()"
+            >
+                <mat-icon>chevron_left</mat-icon>
+            </button>
+            <div class="cards">
+                <app-card
+                    *ngFor="let card of productsGroup"
+                    class="card"
+                    [product]="card"
+                    (buy)="onProductBuy($event)"
+                ></app-card>
+            </div>
+            <button
+                mat-icon-button
+                color="primary"
+                type="button"
+                class="pagination__button-next"
+                (click)="next(); $event.stopPropagation()"
+            >
+                <mat-icon>chevron_right</mat-icon>
+            </button>
+        </div>
+        <div class="pagination">
+            <!-- Add pagination control -->
+            <button
+                *ngFor="let button of pageIndex; let index = index"
+                mat-mini-fab
+                color="primary"
+                type="button"
+                class="pagination__button"
+                [ngClass]="{'is-active': selectIndex === index}"
```

Comment:
Я рекомендую использовать `binding` конкретных классов, на мой взгляд читается намног опроще
```suggestion
                [class.is-active]="selectIndex === index"
```

Reply:
Блин, просто мы почти всегда на работе используем в таком случае ngClass, но могу поправить

Reply:
Повод предложить на работе новую конвенцию) Или спросить, почему именно так используется